.github/workflows/build-x86-image.yaml

```diff
@@ -57,14 +94,43 @@ jobs:
           make ut
 
       - name: Install gosec
-        run: curl -sfL https://raw.githubusercontent.com/securego/gosec/master/install.sh | sh -s -- -b $(go env GOPATH)/bin 'v${{ env.GOSEC_VERSION }}'
+        run: |
+          curl -sfL https://raw.githubusercontent.com/securego/gosec/master/install.sh | \
+            sed 's#giturl=.*#giturl="https://api.github.com/repos/${owner_repo}/releases/tags/${version}"#' | \
+            sed 's#tag_name":"#tag_name": *"#' | \
+            sh -s -- -b $(go env GOPATH)/bin 'v${{ env.GOSEC_VERSION }}'
+
+      - name: Download variable file
+        uses: actions/download-artifact@v3
+        with:
+          name: variables
+
+      - name: Export passed variables
+        run: cat .CI_PASSED_VAR >> "$GITHUB_ENV"
+
+      - name: Download base images
+        if: env.BUILD_BASE == 1
+        uses: actions/download-artifact@v3
+        with:
+          name: kube-ovn-base
+
+      - name: Load base images
+        if: env.BUILD_BASE == 1
+        run: docker load --input image-amd64.tar
 
       - name: Build
         run: |
           go mod tidy
           git diff --exit-code
           make lint
-          make image-kube-ovn
+          if [ "x${{ env.BUILD_BASE }}" = "x1" ]; then
+            TAG=$(cat VERSION)
+            docker tag kubeovn/kube-ovn-base:$TAG-amd64 kubeovn/kube-ovn-base:$TAG
+            docker tag kubeovn/kube-ovn-base:$TAG-amd64-no-avx512 kubeovn/kube-ovn-base:$TAG-no-avx512
+            make build-kube-ovn
+          else
+            make image-kube-ovn
```

Comment:
On Mac the docker buildx will always try to retrieve the remote base image, not sure if the behavior will change if runs on Linux.

Reply:
`build-kube-ovn` is a newly added makefile target, and it does not use buildx.